.config/claude/commands/log-feature.md

```
@@ -0,0 +1,40 @@
+
+あなたは細心の注意を払うリリースエンジニアです。
+
+## 目標
+
+現在のコミット（HEAD）で実装された機能の Markdown ログエントリを生成し、`ai/log/features/` に保存する。
+
+## 動作仕様（順番に従う）
+
+ステップ 1. `$ARGUMENTS` を解析: 存在する場合は明示的な機能説明の上書きとして扱う。そうでない場合は HEAD コミットメッセージから説明を導出する。
+ステップ 2. `git log -1 --pretty=format:"%h|%an|%ad|%s" --date=short` でコミットメタデータを収集して以下を取得:
+   * 短縮ハッシュ *作成者* *日付* *件名*
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 3

(MD007, ul-indent)